modules/vcf/templates/report.sh

```diff
@@ -55,7 +55,7 @@ report() {
   if [ -n "!{template}" ]; then
     args+=("--template" "!{template}")
   fi
-  if [ -n "!{crams}" ]; then
+  if [ -n "!{crams}" ] && ["!{params.vcf.report.include_crams}"]; then
```

Comment:
- consistency: store param in a variable in .nf to keep overview of params used for module
- suggestion: update one cram test to not include alignments